--- origsrc/gimp-2.8.8/build/windows/gimprc-plug-ins.rule	2012-03-12 14:18:08.000000000 -0500
+++ src/gimp-2.8.8/build/windows/gimprc-plug-ins.rule	2013-11-29 01:53:51.932328900 -0600
@@ -3,7 +3,7 @@
 GIMPPLUGINRC = $(top_srcdir)/build/windows/gimp-plug-ins.rc
 
 %.rc.o:
-	$(WINDRES) --define ORIGINALFILENAME_STR="$*$(EXEEXT)" \
+	$(AM_V_GEN)$(WINDRES) --define ORIGINALFILENAME_STR="$*$(EXEEXT)" \
 		--define INTERNALNAME_STR="$*" \
 		--define TOP_SRCDIR="$(top_srcdir)" \
 		-I$(top_srcdir)/app \
--- origsrc/gimp-2.8.8/build/windows/gimprc.rule	2012-03-12 14:18:08.000000000 -0500
+++ src/gimp-2.8.8/build/windows/gimprc.rule	2013-11-29 01:54:14.168600800 -0600
@@ -3,7 +3,7 @@
 GIMPAPPRC = $(top_srcdir)/build/windows/gimp.rc
 
 %.rc.o: 
-	$(WINDRES) --define ORIGINALFILENAME_STR="$*$(EXEEXT)" \
+	$(AM_V_GEN)$(WINDRES) --define ORIGINALFILENAME_STR="$*$(EXEEXT)" \
 		--define INTERNALNAME_STR="$*" \
 		--define TOP_SRCDIR="$(top_srcdir)" \
 		-I$(top_srcdir)/app \
--- origsrc/gimp-2.8.8/libgimp/Makefile.am	2012-11-07 16:53:35.000000000 -0600
+++ src/gimp-2.8.8/libgimp/Makefile.am	2013-11-28 21:25:29.674332500 -0600
@@ -19,14 +19,14 @@ endif
 if OS_WIN32
 gimp_def = gimp.def
 gimpui_def = gimpui.def
-libgimp_export_symbols = -export-symbols gimp.def
-libgimpui_export_symbols = -export-symbols gimpui.def
+libgimp_export_symbols = -export-symbols $(srcdir)/gimp.def
+libgimpui_export_symbols = -export-symbols $(srcdir)/gimpui.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimp-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
 	$(INSTALL) .libs/libgimpui-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimp.def $(DESTDIR)$(libdir)
-	$(INSTALL) gimpui.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimp.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpui.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimp-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpbase/Makefile.am	2013-07-12 08:21:17.000000000 -0500
+++ src/gimp-2.8.8/libgimpbase/Makefile.am	2013-11-28 21:24:16.406141800 -0600
@@ -12,11 +12,11 @@ endif
 if OS_WIN32
 ole32_lib = -lole32
 gimpbase_def = gimpbase.def
-libgimpbase_export_symbols = -export-symbols gimpbase.def
+libgimpbase_export_symbols = -export-symbols $(srcdir)/gimpbase.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpbase-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpbase.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpbase.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpbase-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpcolor/Makefile.am	2012-05-08 01:05:46.000000000 -0500
+++ src/gimp-2.8.8/libgimpcolor/Makefile.am	2013-11-28 21:25:48.236394100 -0600
@@ -13,11 +13,11 @@ endif
 
 if OS_WIN32
 gimpcolor_def = gimpcolor.def
-libgimpcolor_export_symbols = -export-symbols gimpcolor.def
+libgimpcolor_export_symbols = -export-symbols $(srcdir)/gimpcolor.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpcolor-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpcolor.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpcolor.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpcolor-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpconfig/Makefile.am	2012-05-08 01:05:46.000000000 -0500
+++ src/gimp-2.8.8/libgimpconfig/Makefile.am	2013-11-28 21:25:58.193963700 -0600
@@ -10,11 +10,11 @@ endif
 
 if OS_WIN32
 gimpconfig_def = gimpconfig.def
-libgimpconfig_export_symbols = -export-symbols gimpconfig.def
+libgimpconfig_export_symbols = -export-symbols $(srcdir)/gimpconfig.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpconfig-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpconfig.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpconfig.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpconfig-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpmath/Makefile.am	2012-05-08 01:05:46.000000000 -0500
+++ src/gimp-2.8.8/libgimpmath/Makefile.am	2013-11-28 21:26:16.875032200 -0600
@@ -13,11 +13,11 @@ endif
 
 if OS_WIN32
 gimpmath_def = gimpmath.def
-libgimpmath_export_symbols = -export-symbols gimpmath.def
+libgimpmath_export_symbols = -export-symbols $(srcdir)/gimpmath.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpmath-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpmath.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpmath.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpmath-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpmodule/Makefile.am	2012-05-08 01:05:46.000000000 -0500
+++ src/gimp-2.8.8/libgimpmodule/Makefile.am	2013-11-28 21:26:28.615703700 -0600
@@ -8,11 +8,11 @@ endif
 
 if OS_WIN32
 gimpmodule_def = gimpmodule.def
-libgimpmodule_export_symbols = -export-symbols gimpmodule.def
+libgimpmodule_export_symbols = -export-symbols $(srcdir)/gimpmodule.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpmodule-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpmodule.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpmodule.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpmodule-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpthumb/Makefile.am	2013-07-01 12:04:26.000000000 -0500
+++ src/gimp-2.8.8/libgimpthumb/Makefile.am	2013-11-28 21:26:37.798228900 -0600
@@ -11,11 +11,11 @@ endif
 
 if OS_WIN32
 gimpthumb_def = gimpthumb.def
-libgimpthumb_export_symbols = -export-symbols gimpthumb.def
+libgimpthumb_export_symbols = -export-symbols $(srcdir)/gimpthumb.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpthumb-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpthumb.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpthumb.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpthumb-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/libgimpwidgets/Makefile.am	2012-05-08 01:05:46.000000000 -0500
+++ src/gimp-2.8.8/libgimpwidgets/Makefile.am	2013-11-28 21:26:45.118647600 -0600
@@ -15,11 +15,11 @@ endif
 
 if OS_WIN32
 gimpwidgets_def = gimpwidgets.def
-libgimpwidgets_export_symbols = -export-symbols gimpwidgets.def
+libgimpwidgets_export_symbols = -export-symbols $(srcdir)/gimpwidgets.def
 
 install-libtool-import-lib:
 	$(INSTALL) .libs/libgimpwidgets-$(GIMP_API_VERSION).dll.a $(DESTDIR)$(libdir)
-	$(INSTALL) gimpwidgets.def $(DESTDIR)$(libdir)
+	$(INSTALL) $(srcdir)/gimpwidgets.def $(DESTDIR)$(libdir)
 
 uninstall-libtool-import-lib:
 	-rm $(DESTDIR)$(libdir)/libgimpwidgets-$(GIMP_API_VERSION).dll.a
--- origsrc/gimp-2.8.8/plug-ins/win-snap/Makefile.am	2012-05-08 01:05:46.000000000 -0500
+++ src/gimp-2.8.8/plug-ins/win-snap/Makefile.am	2013-11-28 23:01:04.130324600 -0600
@@ -44,4 +44,4 @@ LDADD = \
 	$(INTLLIBS)
 
 winsnap-win32res.o: winsnap.rc select.cur small.ico winsnap.ico
-	$(WINDRES) winsnap.rc winsnap-win32res.o
+	$(AM_V_GEN)$(WINDRES) -I $(srcdir) $(srcdir)/winsnap.rc winsnap-win32res.o
